core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp

```diff
@@ -219,6 +219,12 @@
                         "roleType": "HIVEMETASTORE"
                     }
                 ],
+                "serviceConfigs": [
+                    {
+                        "name": "hive_metastore_enable_ldap_auth",
```

Comment:
should it be in the configprovider?

Reply:
no, it should not, it is fixed for sdx and could not be set by user

Reply:
we are setting hive_metastore_ldap_uri in the configprovider which means every cluster will use this property

Reply:
Yes, but we don't enable the feature.

Reply:
we don't know is it a DL or not, here: https://github.com/hortonworks/cloudbreak/pull/6526/files#diff-3ba4d7ee120408de64e69ad4097ed9da